Bylaws.md

```diff
@@ -127,7 +136,9 @@ This section applies to any ballot that proposes a Final Guideline or a Final Ma
 
 ## 3.1 Associate Members
 
-The Forum may enter into associate member relationships with other organizations when the CA/Browser Forum determines that maintaining such a relationship will be of benefit to the work of the Forum. In the past, entities qualifying as Associate Members have included the ACAB’c, AICPA/CICA WebTrust Task Force, the European Telecommunications Standards Institute (ETSI), Paypal, the Internet Corporation for Assigned Names and Numbers, tScheme, the U.S. Federal PKI, and Certificate Issuers applying for membership but awaiting full qualification under Section 2.1. Participation as an Associate Member is by invitation only. In order to become an Associate Member, an organization must sign a mutual letter of intent, understanding, or other agreement and the Forum’s IPR Policy Agreement, unless this latter requirement is waived in writing by the Forum based on overriding policies of the Associate Member’s own organization IPR rules. Associate Members may attend face-to-face meetings, communicate with Voting Members on Member Mail List, and access Member Web Site. Associate Members are not entitled to vote except on special straw polls of the Forum (e.g. when selecting meeting dates, locations, etc.)
+Membership in the Forum or CWGs as an Associate Member may be by invitation when the CA/Browser Forum determines that maintaining such a relationship will be of benefit to the work of the Forum. In the past, entities qualifying as Associate Members have included the ACAB'c, AICPA/CICA WebTrust Task Force, the European Telecommunications Standards Institute (ETSI), the Internet Corporation for Assigned Names and Numbers, tScheme and the U.S. Federal PKI. In order to become an Associate Member, an organization must sign a mutual letter of intent/understanding, or other similar agreement, and the Forum's IPR Policy Agreement, unless this latter requirement is waived in writing by the Forum based on overriding policies of the Associate Member's own organization IPR rules.
+
+Associate Members may attend face-to-face meetings, communicate with other Members on the Member Mail List and/or Public Mail List, and access the Member Wiki Website. Associate Members are not entitled to vote except on special straw polls of the Forum or CWGs (e.g. when selecting meeting dates, locations, etc.).
```

Comment:
Since “voting” has specific requirements for ballots in Section 2.3 and Section 4.1, should we define what “special straw polls” are and how they are conducted since this activity includes “votes” from Associate Members?

We should also clarify if Associate Members can propose or endorse ballots, using similar language from Section 3.3 Probationary Member.

Reply:
Regarding the first comment about "special straw polls", I don't think we need to make it more explicit. It's ok to have some flexibility for when the Forum asks the opinion of Associate Members or Interested Parties. The critical part is proposing, endorsing and voting for ballot for which I don't think they are allowed to do that.

Regarding the second comment, sure, we can be more explicit and use similar language as in section 3.3 for Associated Members. I will proceed with the following update to the last paragraph of section 3.1:

`Associate Members may not propose or endorse ballots or take part in any form of voting except on special straw polls of the Forum or CWGs (e.g. when selecting meeting dates, locations, etc.).`

I hope this works. Please let me know if you have other suggestions.

Reply:
LGTM